09_simulations/models.py

```diff
@@ -280,142 +273,3 @@ def train_linear_csd(X_train,
     return cv_pipeline
 
 
-def get_prob_metrics(y_train, y_test, y_pred_train, y_pred_test):
```

Comment:
I just wanted to keep this file as small as it made sense to. In my mind `models.py` just has the code for fitting the models and making predictions given some train/test data, and `fit_models.py` has the code for doing everything "around" the model fitting without actually fitting them: e.g. preprocess/split the data, and post-process the predictions by calculating performance metrics using the true labels.

I'll add a bit more detail to the file docstrings about this (and maybe change the file names since I now realize `fit_models.py` doesn't actually fit the models).